CONTRIBUTING

```diff
@@ -1 +0,0 @@
-Please see http://docs.php-http.org/en/latest/development/contributing.html
```

Comment:
This means that GitHub does not show this "warning" anymore. 
<img width="1051" alt="screen shot 2016-07-16 at 12 02 30" src="https://cloud.githubusercontent.com/assets/1275206/16894117/32a55f02-4b4d-11e6-9622-0142976db4a4.png">